cpp/roaring64map.hh

```diff
@@ -826,7 +826,7 @@ public:
     Roaring64MapSetBitForwardIterator(const Roaring64Map& parent, bool exhausted = false)
         : map_end(parent.roarings.cend())
     {
-        if(exhausted) {
+        if(exhausted || parent.roarings.empty()) {
```

Comment:
An empty map needs special handling to create an exhausted iterator or this constructor will attempt to extract the first Roaring.